src/blog/wasm-compilation-pipeline.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'WebAssembly Compilation Pipeline'
+author: 'Andreas Haas, compiler of modules'
+avatars:
+  - andreas-haas
+date: 2020-11-05
+tags:
+  - WebAssembly
+  - internals
+description: 'This article explains V8’s WebAssembly compilers and when they compile WebAssembly code.'
+tweet: 'XXXXXXXXXX'
```

Comment:
```suggestion
tweet: ''
```

if it's non-empty, it will generate and render a link, which would be invalid in this case

Reply:
Done.